pkg/constants/constants.go

```diff
@@ -195,10 +195,12 @@ func GetInstallKeyEnvMap() map[string]string {
 // GetResetKeyEnvMap returns environment variable bindings to ResetSpec data
 func GetResetKeyEnvMap() map[string]string {
 	return map[string]string{
-		"target":          "TARGET",
-		"system.uri":      "SYSTEM",
-		"tty":             "TTY",
-		"grub-entry-name": "GRUB_ENTRY_NAME",
+		"system.uri":       "SYSTEM",
+		"tty":              "TTY",
+		"grub-entry-name":  "GRUB_ENTRY_NAME",
+		"cloud-init":       "CLOUD_INIT",
+		"reset-persistent": "PERSISTENT",
+		"reset-oem":        "OEM",
```

Comment:
These will translate in to "ELEMENTAL_RESET_*" variables. For instance `reset-oem` flag will be mapped as `ELEMENTAL_RESET_OEM` variable and `cloud-init` to `ELEMENTAL_RESET_CLOUD_INIT`, etc.